python/runtime/local/xgboost_submitter/predict.py

```diff
@@ -117,8 +118,10 @@ def _calc_predict_result(bst, dpred, model_params):
     objective = model_params.get("objective", "")
     if objective.startswith("binary:"):
         preds = (preds > 0.5).astype(np.int64)
-    elif objective.startswith("multi:") and len(preds) == 2:
-        preds = np.argmax(np.array(preds), axis=1)
+    elif objective.startswith("multi:") and preds.ndim == 2:
```

Comment:
Should check the objective equals `"multi:softprob"` to run `argmax` since `multi:softmax` outputs label and `multi:softprob` outputs prob. Please checkout `runtime.xgboost.predict`.

Reply:
- `multi:softmax` outputs label so that `preds.ndim == 1`.
- `multi:softprob` outputs prob so that `preds.ndim == 2`.

Here, if `preds.ndim == 2`, we think that the result is the prob and we should apply `argmax`; if `preds.ndim == 1`, we think that the result is the label and we do not apply `argmax`.